paparazzi/paparazzi-gradle-plugin/build.gradle

```diff
@@ -11,17 +11,28 @@ gradlePlugin {
   }
 }
 
+configurations {
+  fixtureClasspath
+}
+
+tasks.named('pluginUnderTestMetadata').configure {
+  getPluginClasspath().from(configurations.fixtureClasspath)
+}
+
 dependencies {
   compileOnly gradleApi()
-  implementation platform(libs.kotlin.bom)
   implementation libs.plugin.kotlin
-  implementation libs.plugin.android
```

Comment:
please merge this PR, I can't use Android Studio Dolphin with 1.2.0-SNAPSHOT, because it doesn't support 7.4:
> The project is using an incompatible version (AGP 7.4.0-rc01) of the Android Gradle plugin. Latest supported version is AGP 7.3.1
> [See Android Studio & AGP compatibility options.](https://developer.android.com/studio/releases#android_gradle_plugin_and_android_studio_compatibility)

I'm not even sure of a workaround yet, because I'm using `plugins {}` so there's no `classpath(...) { exclude }`.

Reply:
Tried this, but it doesn't apply transitively and I don't want to list all modules:
```gradle
buildscript {
	configurations.classpath.configure {
		resolutionStrategy { 
			force(libs.android.gradle)
			force(libs.android.lint.common)
    }
```

This crazy hack works 🤦‍♂️:
```gradle
buildscript {
	configurations.classpath.configure {
		resolutionStrategy { 
			eachDependency { 
				if (requested.version == "30.4.0-rc01") {
					useVersion("30.3.1")
				}
				if (requested.version == "7.4.0-rc01") {
					useVersion("7.3.1")
				}
			}
		}
	}
}
```